criu/mount.c

```diff
@@ -2734,6 +2736,13 @@ struct mount_info *mnt_entry_alloc()
 
 	new = xzalloc(sizeof(struct mount_info));
 	if (new) {
+		if (rst) {
+			new->remounted_rw = shmalloc(sizeof(int));
```

Comment:
Shouldn't we free `remounted_rw` in `mnt_entry_free()` ?

Reply:
No we can not, with shmalloc we can only free last allocation (see shfree_last) so in general we can't free shmalloc'ed data.

Reply:
Oof okay I see. I thought from the name that "last" meant that the user should do reference counting or something.
shfree_last() shouldn't take an argument. I think I wouldn't have been confused if it was just `shfree_last()`